tools/ogs_tools/shp2msh.xml

```diff
@@ -0,0 +1,48 @@
+<tool id="shp2msh" name="shp2msh_converter" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
+    <requirements>
+        <requirement type="package" version="0.0.3">ogstools</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        export SHAPE_RESTORE_SHX=YES &&    
+        ln -s '$input_shp' input.shp &&
+        cat input.shx > /home/stehling/ogs_tools/content_of_input.txt &&
+        shp2msh --input input.shp --output output.vtu
```

Comment:
Should the output file be named .msh (to avoid confusion) -- in case output extensions do not matter you can also try to 

```suggestion
        shp2msh --input input.shp --output '$output'
```

then you do not need the `from_workdir`